goss-testing/suites/ncn-kubernetes-tests-master-single.yaml

```diff
@@ -34,6 +34,7 @@
 # corresponding suite file without the -single suffix.
 gossfile:
   ../tests/goss-ceph-csi-k8s-requirements.yaml: {}
+  ../tests/goss-compare-k8s-ncns.yaml: {}
```

Comment:
Why did you choose to add it to this suite? Is this suite run after all nodes have been upgraded and never between node upgrades? I see there is a suite that is run [after csm-upgrades](https://github.com/Cray-HPE/csm-testing/blob/2d629b5f632d22b99d0f2d018aabe95c5383571f/goss-testing/automated/run-ncn-tests.sh#L551)

Reply:
I could also add it to the suite for after csm upgrades, but I also wanted it to be part of the regular CSM health checks. That way it would catch other cases where the nodes ended up mismatched (for example, if a node was rebuilt using the wrong image).

I don't think the suite I added it to is run during CSM upgrades, but I will double check now.

Reply:
I searched the `cray-nls` and `docs-csm` repos for all places where this suite is called, and there are no cases where it is called during an upgrade. It is only run during normal CSM health checks (which are supposed to be done before and after upgrades).

Reply:
Okay, makes sense. Thanks for looking into it!